apollo-ast/api/apollo-ast.api

```diff
@@ -1121,6 +1121,7 @@ public final class com/apollographql/apollo3/ast/Schema {
 	public static final field REQUIRES_OPT_IN Ljava/lang/String;
 	public static final field TYPE_POLICY Ljava/lang/String;
 	public fun <init> (Ljava/util/List;)V
+	public final fun getConnectionTypes ()Ljava/util/Set;
```

Comment:
can you make this `@ApolloInternal`? I'd like to make `Schema` (and apollo-ast) generic enough that it doesn't expose the Apollo specific parts. `foreignNames` is already an exception to this rule but it's not reason not to do it for this one while we can.